main.py

```
@@ -0,0 +1,481 @@
+            "limit": limit,
+            "total": 100
+        }
+    }
+
+
+# 회원 가입
+@app.post("/users", response_model=user.ResponseUser, status_code=201)
+async def post_users(
+        user_data: Annotated[user.CreateUser, Body()]
+):
+    current_time = datetime.now().isoformat()
```

Ruff (0.14.13)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)